ui/packages/admin-security-certificate-ui/src/main/webapp/js/model/UrlCertificate.js

```diff
@@ -26,11 +26,11 @@ define([
         });
 
         UrlCertificate.Response = Backbone.AssociatedModel.extend({
-            url: "../jolokia/exec/org.codice.ddf.security.certificate.keystore.editor.KeystoreEditor:service=keystore/showCertificateFromUrl",
+            url: "../jolokia/exec/org.codice.ddf.security.certificate.keystore.editor.KeystoreEditor:service=keystore/certificateDetails",
             saveUrl: "../jolokia/exec/org.codice.ddf.security.certificate.keystore.editor.KeystoreEditor:service=keystore/addTrustedCertificateFromUrl",
             relations: [
                 {
-                    type: Backbone.Many,
+                    type: Backbone.One,
```

Comment:
❓ Odd that this was Many before, but definitely agree that if it's a single value we should change it to One.

Reply:
👍